### PR TITLE
feat(kmesh): UBI images

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Docs about upcoming UBI images distributions. It will be released with the next release of Kong Mesh.

### CODE REVIEW

File: app/mesh/1.5.x/features/ubi-images.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+helm install kuma \
+  --namespace kuma-system \
+  --set kuma.controlPlane.image.repository=ubi-kuma-cp \
+  --set kuma.dataPlane.image.repository=ubi-kuma-dp \
+  --set kuma.dataPlane.image.repository=ubi-kuma-dp \
+  --set kuma.dataPlane.initImage.repository=ubi-kuma-dp \
+  --set kuma.kumactl.image.repository=ubi-kumactl \
+  kuma/kuma
+```
+{% endnavtabs %}
```
Comment (lena-larionova) [inline app/mesh/1.5.x/features/ubi-images.md:41]:
```suggestion
{% endnavtab %}
{% endnavtabs %}
```
Missing a closing tag here, it's why the build is breaking.

File: app/mesh/1.5.x/features/ubi-images.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+{% navtabs %}
+{% navtab kumactl %}
+```sh
+kumactl install control plane \
+  --control-plane-repository=ubi-kuma-cp \
+  --dataplane-init-repository=ubi-kuma-init \
+  --dataplane-repository=ubi-kuma-dp
+```
+{% endnavtab %}
+{% navtab HELM %}
```
Comment (lena-larionova) [inline app/mesh/1.5.x/features/ubi-images.md:30]:
```suggestion
{% navtab Helm %}
```

**Comment by mheap** [comment]
Broken link checker is because the branch is not up to date with `main`. All passing otherwise

**Comment by mheap** [comment]
@jakubdyszkiewicz When will the UBI images be available?

**Comment by jakubdyszkiewicz** [comment]
@mheap UBI images will be available in upcoming 1.6 release (next week)

**Comment by lahabana** [comment]
@jakubdyszkiewicz can you move your change to 1.6 please?

File: app/mesh/1.6.x/features/ubi-images.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Red Hat Universal Base Images
+---
+
+In addition to the standard Kong Mesh images build on Alipne Linux, Kong Mesh also ships with images based on [Red Hat Universal Base Image (UBI)](https://developers.redhat.com/products/rhel/ubi).
```
Comment (lena-larionova) [inline app/mesh/1.6.x/features/ubi-images.md:5]:
```suggestion
In addition to the standard Kong Mesh images built on Alpine Linux, Kong Mesh also ships with images based on the [Red Hat Universal Base Image (UBI)](https://developers.redhat.com/products/rhel/ubi).
```

File: app/mesh/1.6.x/features/ubi-images.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Kong Mesh UBI images are distributed with all standard images, but with `ubi-` prefix.
```
Comment (lena-larionova) [inline app/mesh/1.6.x/features/ubi-images.md:7]:
```suggestion
Kong Mesh UBI images are distributed with all standard images, but with the `ubi-` prefix:
```

File: app/mesh/1.6.x/features/ubi-images.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+* [kuma-cp UBI Image](https://hub.docker.com/r/kong/ubi-kuma-cp)
+* [kumactl UBI Image](https://hub.docker.com/r/kong/ubi-kumactl)
+* [kuma-init UBI Image](https://hub.docker.com/r/kong/ubi-kuma-init)
+* [kuma-prometheus-sd UBI Image](https://hub.docker.com/r/kong/ubi-kuma-prometheus-sd)
+
+The base UBI variant for all images is `ubi-minimal`.
+
+## Usage
+
+To use UBI images, we need to explicitly pass them when the control plane is installed
```
Comment (lena-larionova) [inline app/mesh/1.6.x/features/ubi-images.md:19]:
```suggestion
To use UBI images, you need to explicitly pass them when the control plane is installed:
```

**Review by lena-larionova — COMMENTED** [review]
Minor feedback that got lost in the move to 1.6.x.

**Pull request merged**